### PR TITLE
Remove image id_minter

### PR DESCRIPTION
**Pull request by alicefuzier**

### CODE REVIEW

File: pipeline/terraform/ecr.tf [hunk 1]
```diff
@@ -18,12 +18,8 @@ resource "aws_ecr_repository" "ecr_repository_transformer_calm" {
   name = "uk.ac.wellcome/transformer_calm"
 }
 
-resource "aws_ecr_repository" "ecr_repository_id_minter_works" {
-  name = "uk.ac.wellcome/id_minter_works"
-}
-
-resource "aws_ecr_repository" "ecr_repository_id_minter_images" {
-  name = "uk.ac.wellcome/id_minter_images"
```
Comment (alexwlchan) [inline pipeline/terraform/ecr.tf:26]: Is this going to break the existing prod pipeline when you apply it? Services won't be able to pull images.

Comment (alicefuzier) [reply pipeline/terraform/ecr.tf:26]: argh, yes. Thank you good spot

**Review by kenoir — APPROVED** [review]
Looks good, bar @alexwlchan's comment about the ECR repo.

File: pipeline/terraform/ecr.tf (hunk 1 above)
Comment (alexwlchan) [reply pipeline/terraform/ecr.tf:26]: Yeah, I’m also wondering if changing the order of services/messaging will break the pipeline?

Comment (alicefuzier) [reply pipeline/terraform/ecr.tf:26]: I hope not. The same thing was done to move the id minter from after the relation embedder to before. We have to target the apply but other than that it shouldn't have an impact on prod? (Also, the changing of order is already merged in master. this PR is just deleting the image id minter)

**Pull request merged**